frontend/packages/kubevirt-plugin/src/utils/validations/vm/vm.ts

```diff
@@ -16,14 +40,34 @@ export const validateVmLikeEntityName = (
     existsErrorMessage: undefined,
     subject: undefined,
   },
+  validations: UIValidation[] = undefined,
 ): ValidationObject => {
   const dnsValidation = validateDNS1123SubdomainValue(value, { subject });
-  return dnsValidation && dnsValidation.type === ValidationErrorType.Error
-    ? dnsValidation
-    : validateEntityAlreadyExists(value, namespace, vmLikeEntities, {
-        errorMessage: existsErrorMessage,
-        subject,
-      });
+  const dnsValidationFailed = dnsValidation?.type === ValidationErrorType.Error;
+  const lengthValidation = validations?.[0]
+    ? validateUIValidationType(validations?.[0], value)
+    : null;
+  const lengthValidationFailed = lengthValidation?.type === ValidationErrorType.Error;
+
+  let validation = dnsValidation;
+  if (lengthValidationFailed && dnsValidationFailed) {
```

Comment:
I am sorry, but this all seems hacky. Can't we just add  optional `min` and `max` values to validateDNS1123SubdomainValue? This would make the code much simpler I think.

Reply:
No need to apologize. I can definitely do that, but I'm not sure how it will allow me to clean this up much more. Since `validateDNS1123SubdomainValue` validates more than just length, we have to account for multiple scenarios. If it only validated length, the logic could be made much clearer. Perhaps I'm just not seeing a better way. Could you give me an idea of what you're thinking?

Reply:
It can only validate what it is validating already plus simple min/max params. No need for other scenarios. Let's not focus on future `UIValidation`s 
It will help us not having to dissect returned messages.

Reply:
I was trying to avoid touching code outside of kubevirt, but that would make it much simpler.

Reply:
Sorry for the confusion. Done!